stable/datadog/templates/container-agent.yaml

```diff
@@ -81,6 +81,8 @@
     - name: DD_KUBERNETES_POD_ANNOTATIONS_AS_TAGS
       value: '{{ toJson .Values.datadog.podAnnotationsAsTags }}'
     {{- end }}
+    - name: DD_APM_ENABLED
+      value: {{ default false .Values.datadog.apm.enabled | quote }}
```

Comment:
here the value should be always "false", else you will have again the trace-agent running in the agent container, and at the same time it will run in the trace-agent container.

Reply:
What do you mean? This just the environment variable. However I'm not sure if this fixes the problem...

Reply:
We have several init script that use environment variable to configure the agents